### ISSUE TITLE
Portrait mode level property is completely messed up

### ISSUE THREAD

**Issue by rollerozxa** [Bithack/principia#141]
##### Principia version
master

##### OS / Hardware
- Platform: Android

##### Summary
Portrait mode level property is completely messed up.

##### Steps to reproduce
Enable portrait mode in the sandbox or try [Oliback's pinball level](https://archive.principia-web.se/level/3251).

![image](https://github.com/Bithack/principia/assets/60856959/624be8fc-9d38-4cab-9338-93d56ce09379)

**Comment by griffi-gh** [comment]
tbh i think it should be dropped in favor of real portrait mode

**Comment by rollerozxa** [comment]
That's what it used to do, I think. Unsure when things broke.

**Comment by griffi-gh** [comment]
> That's what it used to do, I think. Unsure when things broke.

nope, it literally just forces the ui into a portrait-like shape. it's called emulate portrait mode for a reason

**Comment by rollerozxa** [comment]
![image](https://github.com/Bithack/principia/assets/60856959/2a939df1-154c-4586-a6f2-212d474b931f)

**Comment by griffi-gh** [comment]
well the hamburger menu is supposed to be centered like that afaik (for convinience)
